README.md

```diff
@@ -385,10 +387,10 @@ M-x ts-fold-indicators-mode
 Please note that turning on `ts-fold-indicators-mode` automatically turns on
 `ts-fold-mode` as well.
 
-- To enable this automatically whenever `tree-sitter-mode` is enabled:
+- To enable this automatically whenever `tree-sitter-mode` is enabled, use the global indicator mode:
 
   ```elisp
-  (add-hook 'tree-sitter-after-on-hook #ts-fold-indicators-mode)
+  (global-ts-fold-indicators-mode 1)
```

Comment:
I prefer to keep both versions and let users decide what to do.

Reply:
Sounds good, will update so both are documented.